### PR TITLE
luci-app-statistics: add support for cUrl

### PR DESCRIPTION
**Pull request by jinchizhong**
Add collectd-mod-curl plugin support. Which can do some complex track,
such as grab stock, but by now, only response time are supported.

#2138

### CODE REVIEW

File: applications/luci-app-statistics/root/etc/config/luci_statistics [hunk 1]
```diff
@@ -66,6 +66,9 @@ config statistics 'collectd_cpu'
 config statistics 'collectd_cpufreq'
 	option enable '0'
 
+config statistics 'collectd_curl'
+    option enable '0'
```
Comment (hnyman) [inline applications/luci-app-statistics/root/etc/config/luci_statistics]: minor nitpick: tab instead of spaces

Comment (hnyman) [reply applications/luci-app-statistics/root/etc/config/luci_statistics]: I think I will fix it myself before committing this.

**Comment by hnyman** [comment]
Apart from that one tab change, the code compiles and could be merged.
But I just noticed that you have not "signed-off-by" the commit. That is required.

Please amend your commit in the PR:
That one tab change, plus add the signed-off-by line into the commit message itself.

**Comment by jinchizhong** [comment]
Hi @hnyman 

I have rewrite my commit to add signed-off-by.
And fixed the space/tab problem.

**Pull request merged**